src/ruby/README.md

```diff
@@ -48,7 +48,7 @@ $ gem install bundler
 ```sh
 $ # from this directory
 $ bundle install  # creates the ruby bundle, including building the grpc extension
-$ rake  # runs the unit tests, see rake -T for other options
+$ CONFIG='specify the directory of `protc` and `grpc_ruby_plugin` command' rake  # runs the unit tests, see rake -T for other options
```

Comment:
rather than "directory" of these commands, this should say something more like "build config subdirectory". 

Can we note an example here, e.g. "`make grpc_ruby_plugin`" by default will place binaries in the  `opt` build config subdirectory and `CONFIG=dbg make grpc_ruby_plugin` will place the executables under the `dbg` build config subdirectory

```
make grpc_ruby_plugin
```

will e.g. result place the plugin under the `opt` subdirectory,

and that

```
CONFIG=dbg make grpc_ruby_plugin
```

Reply:
OK. Thanks. I changed a description to 'build config subdirectory'.

https://github.com/grpc/grpc/pull/15357/commits/e8d07f25ba9808692f182c635dbd6f8b27444bcd